frontend/src/static/js/main.js

```diff
@@ -169,6 +174,34 @@ window.app = new window.Vue({
         document.getElementById('tabs-wrapper').scrollTop = 0;
       }
     },
+    renderAuthorShipTabHash(minDate, maxDate) {
+      const hash = window.hashParams;
+      const info = {
+        author: hash.tabAuthor,
+        location: hash.tabLocation,
+        minDate,
+        maxDate,
+      };
+      const tabInfoLength = 5;
+      this.updateInfoRepoName(info);
+      if (Object.keys(info).length === tabInfoLength) {
+        this.updateTabAuthorship(info);
+      } else if (hash.tabOpen === 'false') {
+        window.app.isTabActive = false;
+      }
+    },
+    updateInfoRepoName(info) {
+      if (!info.location || info.location.split('github.com/').length < 2) {
+        return;
+      }
+      const repoName = info.location.split('github.com/')[1].split('/');
+      if (repoName.length === 2) {
+        info.repo = `${repoName.join('_').slice(0, -4)}_master`;
```

Comment:
This part doesn't seems to be right... it assumes that the branch would be `master`.

![image](https://user-images.githubusercontent.com/19277206/53345104-20fa8780-394f-11e9-8bec-d492563ec585.png)